src/signals/incident/services/map-values/index.js

```diff
@@ -0,0 +1,42 @@
+import forEach from 'lodash.foreach';
+import set from 'lodash.set';
+import isObject from 'lodash.isobject';
+
+import getControls from '../get-controls';
+
+const convertValue = (value) => {
+  if (value === 0) {
+    return 0;
+  }
+  if (value === true) {
+    return 'ja';
+  }
+  if (value === false) {
+    return 'nee';
+  }
+  if (isObject(value) && value.id) {
+    return value.id;
+  }
+
+  return value;
+};
```

Comment:
`convertValue` is used in two different places. Suggest extracting it to different file and reuse.

Reply:
gedaan